components/GroupFilterModal.js

```diff
@@ -77,16 +77,6 @@ export default ({
       propagateSwipe={true}
       style={styles.modal}
     >
-      <AntDesign.Button
```

Comment:
I think this is unnecessary (and low 🔑  it was in the wrong place--on the right vs the left. Android is right, iOS is left). Since this is a pull up/down tray anyway, which hopefully is clearly indicated by both the experience of the tray sliding up and the big bar at the top, the user should be used to dismissing it without needing the X.

Reply:
i don't think it really matters what side the `x` is on 🤔 
but i'm fine removing it here 👍